### ISSUE TITLE
Create embedded API Playground page

### ISSUE THREAD

**Issue by andrewhercules** [opentargets/platform#1792]
As a **user** I want **visit a page with an embedded GraphQL API playground** because **I want to run sample GraphQL API queries without leaving the Platform**

## Background

Alongside the inclusion of a button that will open the drawer component and allow users to run a GraphQL API query that powers a given data table — see #1750 — we also want to host an embedded version of the GraphQL API playground. This will allow users to run sample queries that they can edit without needing to leave the Platform. It will also help users become familiar with accessing our data in different ways and will include links to other data access touch points for more comprehensive or systematic queries. 

Pharos provides a similar feature on [their API page](https://pharos.nih.gov/api).

## Tasks

- [ ] Investigate GraphQL API playground React component
- [ ] Create new page `/api` with component visible

Further tasks are TBC, pending design specification.

## Acceptance tests

How do we know the task is complete? 

1. When I go visit `platform.opentargets.org/api` I see the embedded GraphQL API playground 
2. On the left-hand side, I see sample queries for: 1) annotation information for a single target, a single disease/phenotype, and a single drug; 2) association information for a single target or disease/phenotype; and 3) evidence for a single datasource for a single target-disease association 
3. I also see links to the API documentation and BigQuery instance
4. When I click on one of the sample queries, the playground updates with the query
5. When I click the "Play" button, the query executes and I see the results in the right-hand side of the playground

**Comment by andrewhercules** [comment]
@mirandaio, below are some API queries that can be used to populate the different examples on the /api page:

Because each section will have only 1 or 2 queries, can we drop the `(_n_)` number beside the query category?

### Target-disease associations 

Label: Find targets associated with a specific disease or phenotype

```
query associatedTargets {
  disease(efoId:"EFO_0000349"){
    id
    name
    associatedTargets{
      count
      rows{
        target{
          id
          approvedSymbol
        }
        score
      }
    }
  }
}
```

Label: Find diseases and phenotypes associated with a specific target

```
query associatedDiseases {
  target(ensemblId:"ENSG00000127318"){
    id
    approvedSymbol
    associatedDiseases{
      count
      rows{
        disease{
          id
          name
        }
        datasourceScores{
          id
          score
        }
      }
    }
  }
}
```

### Target-disease evidence

Label : Explore evidence from that supports a specific target-disease association

```
query targetDiseaseEvidence {
  disease(efoId: "EFO_0005952") {
    id
    name
    evidences(datasourceIds: ["intogen"], ensemblIds: ["ENSG00000172936"]) {
      count
      rows {
        disease {
          id
          name
        }
        diseaseFromSource
        target {
          id
          approvedSymbol
        }
        mutatedSamples {
          functionalConsequence {
            id
            label
          }
          numberSamplesTested
          numberMutatedSamples
        }
        resourceScore
        significantDriverMethods
        cohortId
        cohortShortName
        cohortDescription
      }
    }
  }
}
```

### Target annotation

Label: Find tractability and safety information for a specific target
```
query targetAnnotation{
  target(ensemblId:"ENSG00000169083"){
    id
    approvedSymbol
    tractability{
      modality
      id
      value
    }
    safetyLiabilities {
      event
      eventId
      biosample {
        cellFormat
        cellLabel
        tissueLabel
        tissueId
      }
      effects {
        dosing
        direction
      }
      study {
        name
        type
        description
      }
      datasource
      literature
    }
  }
}
```

### Disease annotation

Label: Find clinical signs and symptoms for a specific disease
```
query diseaseAnnotation {
  disease(efoId: "EFO_0003885") {
    id
    name
    phenotypes {
      rows {
        phenotypeHPO {
          id
          name
          description
          namespace
        }
        phenotypeEFO {
          id
          name
        }
        evidence {
          aspect
          bioCuration
          diseaseFromSourceId
          diseaseFromSource
          evidenceType
          frequency
          frequencyHPO {
            name
            id
          }
          qualifierNot
          onset {
            name
            id
          }
          modifiers {
            name
            id
          }
          references
          sex
          resource
        }
      }
    }
  }
}

```

### Drug annotation 

Label: Find approval status and withdrawn and black-box warning for a specific drug

```
query drugApprovalWithdrawnWarningData {
  drug(chemblId:"CHEMBL118"){
    name
    id
    isApproved
    hasBeenWithdrawn
    blackBoxWarning
    drugWarnings{
      warningType
      description
      toxicityClass
      meddraSocCode
      year
      references{
        id
        source
        url
      }
    }
  }
}
```

**Comment by andrewhercules** [comment]
Ticket closed as work completed, merged into `main`, and released in `21.11`
